lxd/api_metrics.go

```diff
@@ -338,6 +339,28 @@ func internalMetrics(ctx context.Context, daemonStartTime time.Time, tx *db.Clus
 		out.AddSamples(metrics.OperationsTotal, metrics.Sample{Value: float64(len(operations))})
 	}
 
+	instances, err := dbCluster.GetInstances(ctx, tx.Tx())
```

Comment:
This could be quite an expensive db call, can we count the instance metrics we are already generating to avoid more queries?